packages/console/src/components/PlanUsage/PlanUsageCard/index.tsx

```diff
@@ -78,29 +135,83 @@ function PlanUsageCard({
           <Trans
             components={{
               span: (
-                <span className={classNames(styles.usageTip, isUsageTipHidden && styles.hidden)} />
+                <span
+                  className={classNames(
+                    styles.usageTip,
+                    // Hide usage tip for free plan users.
+                    (!isPaidTenant || basicQuota === undefined || hideQuotaNotice) && styles.hidden
+                  )}
+                />
               ),
             }}
           >
-            {t(usageKey, {
-              usage:
-                quota === undefined
-                  ? formatNumber(usage)
-                  : typeof quota === 'number'
-                  ? `${formatNumber(usage)} / ${formatNumber(quota)}${
-                      usagePercent === undefined ? '' : ` (${(usagePercent * 100).toFixed(0)}%)`
-                    }`
-                  : `${formatNumber(usage)} / ${String(t('subscription.quota_table.unlimited'))}`,
-            })}
+            {t(
+              (() => {
+                if (basicQuota === null || basicQuota === true) {
+                  return 'subscription.usage.usage_description_with_unlimited_quota';
+                }
+
+                if (basicQuota === false || basicQuota === 0) {
+                  return 'subscription.usage.usage_description_without_quota';
+                }
+
+                if (typeof basicQuota === 'number') {
+                  return 'subscription.usage.usage_description_with_limited_quota';
+                }
+
+                return usageKey;
+              })(),
+              isPaidTenant
+                ? {
+                    usage: formatNumber(usage),
+                    ...conditional(
+                      typeof basicQuota === 'number' && {
+                        basicQuota: formatQuotaNumber(basicQuota),
+                      }
+                    ),
+                  }
+                : {
+                    usage: formatNumberTypedUsageDescription({
+                      usage,
+                      quota,
+                      unlimitedString: String(t('subscription.quota_table.unlimited')),
+                    }),
+                  }
```

Comment:
nit: Consider using `useMemo`

Reply:
in previous part, we have already narrowed down the type of `usage` to be number. If we use `useMemo` hook, we still need to handle the logic when `usage`'s type is not number. Keep current implementation to make code simple.